core/src/task.rs

```diff
@@ -30,10 +30,7 @@ pub enum VdafInstance {
     /// A vector of `Prio3` sums.
     Prio3SumVec { bits: usize, length: usize },
     /// A `Prio3` histogram.
-    Prio3Histogram {
-        #[derivative(Debug(format_with = "bucket_count"))]
-        buckets: Vec<u64>,
-    },
+    Prio3Histogram { buckets: usize },
```

Comment:
I have a slight preference for naming the parameter `length` instead of `buckets` here, in order to match with what VDAF uses.